src/views/Projects/index.jsx

```diff
@@ -20,11 +21,32 @@ import ProjectCard from '../../components/ProjectCard';
 export default class Projects extends Component {
   state = {
     projects,
+    search: '',
+  };
+  handleTextInputChange = event => {
+    this.setState({ search: event.target.value });
   };
 
   render() {
     const { classes } = this.props;
-    const { projects } = this.state;
+    const { projects, search } = this.state;
+    const selectedProjects = Object.keys(projects)
```

Comment:
We are filtering on projects not necessarily selecting. I think something like `filteredProjects` is clearer.